src/DIRAC/WorkloadManagementSystem/DB/SandboxMetadataDB.py

```diff
@@ -128,7 +128,10 @@ def registerAndGetSandbox(self, owner, ownerGroup, sbSE, sbPFN, size=0):
             if not result["Value"]:
                 return S_ERROR("SandBox already exists but doesn't belong to the user")
             sbId = result["Value"][0][0]
-            self.accessedSandboxById(sbId)
+            if not (
+                ret := self._update(f"UPDATE `sb_SandBoxes` SET LastAccessTime=UTC_TIMESTAMP() WHERE SBId = {sbId}")
+            ):
```

Comment:
```suggestion
            )["OK"]:
```
Does this work? In any case `_update` returns a S_OK/S_ERROR dict.